input/core/src/commonMain/kotlin/majestic/editor/body/chunkUI/IconedSelector.kt

```diff
@@ -0,0 +1,128 @@
+package majestic.editor.body.chunkUI
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ExposedDropdownMenuBox
+import androidx.compose.material3.ExposedDropdownMenuDefaults
+import androidx.compose.material3.MenuAnchorType
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import majestic.editor.insert.Insert
+
+data class BorderColors(
+    val focused: Color,
+    val unfocused: Color
+)
+
+data class TextColors(
+    val focused: Color,
+    val unfocused: Color
+)
+
+data class TextField(
```

Comment:
Calling this class `TextField` is not not a good descision. Find a better name for this class please